documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -2234,3 +2238,83 @@ Eg:
 ```
 
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [ARM](#arm-violations): [Errors](#arm-errors) or [Warnings](#arm-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
+
+### <a name="r4005" ></a>R4005 XmsEnumNameUnique
```

Comment:
if to follow the naming code, I wonder if the rule name should be "UniqueXmsEnumName" instead of "XmsEnumNameUnique"